integration/basic_test.go

```diff
@@ -195,3 +197,59 @@ func TestCollectionName(t *testing.T) {
 		assert.Contains(t, names, longCollectionName)
 	})
 }
+
+func TestDatabaseName(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Err", func(t *testing.T) {
+		ctx, collection := Setup(t)
+
+		dbName300 := strings.Repeat("a", 300)
+
+		cases := map[string]struct {
+			db  string
+			err *mongo.CommandError
+			alt string
+		}{
+			"TooLongForBoth": {
```

Comment:
I don't understand this case name - `TooLongForBoth` - who are those `both`?

Reply:
a little thing but maybe `TooLongForBothDBs` could be more readable :)

Reply:
>  a little thing but maybe TooLongForBothDBs could be more readable :)

Good suggestion changed for collection name and database name tests.